.github/workflows/release.yml

```diff
@@ -15,36 +13,46 @@ env:
   AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
   AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
   KEYS_BUCKET: ${{ secrets.AWS_S3_KEYS_BUCKET }}
+  BUILDS_BUCKET: ${{ secrets.AWS_S3_BUILDS_BUCKET }}
+  KEYCHAIN_NAME: 'fastlane_tmp_keychain'
+  KEYCHAIN_PASSWORD: ''
   # iOS Release - all sensitive values should be stored as repo Secrets
   APPLE_PROFILE: ${{ secrets.APPLE_PROFILE }}               # Filename for the provisioning profile, eg AppStore_comrootstraiosbasedevelop.mobileprovision
+  APPLE_CERT: ${{ secrets.APPLE_CERT }}
+  APPLE_KEY: ${{ secrets.APPLE_KEY }}
```

Comment:
Could we add a comment explaining that these are the filenames of the cert and key files uploaded to S3?